tapestry.php

```diff
@@ -132,7 +137,7 @@ function tapestry_enqueue_vue_app()
     global $post;
     if ('tapestry' == get_post_type($post) && !post_password_required($post)) {
         global $TAPESTRY_VERSION_NUMBER;
-        global $TAPESTRY_USE_DEV_MODE;
+        $TAPESTRY_USE_DEV_MODE = defined('TAPESTRY_USE_DEV_MODE') && !empty(TAPESTRY_USE_DEV_MODE);
 
         $use_dev = $TAPESTRY_USE_DEV_MODE || isset($_GET['debug']);
```

Comment:
I would merge these into a single line since `$TAPESTRY_USE_DEV_MODE` is not used anywhere else and it doesn't add much clarity here to have it in 2 lines.

(not linted)
```suggestion
        $use_dev = (defined('TAPESTRY_USE_DEV_MODE') && !empty(TAPESTRY_USE_DEV_MODE)) || isset($_GET['debug']);
```

Reply:
Got it. This has now been changed.